src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.c

```diff
@@ -904,64 +968,63 @@ void func_80AEB698(EnLiftNuts* this, PlayState* play) {
     }
 }
 
-void func_80AEB828(EnLiftNuts* this) {
-    if (!func_80AE9B4C(0, 4)) {
-        func_80AE9B4C(1, 4);
-        this->unk_356 = 1;
+void EnLiftNuts_SetupStartHiding(EnLiftNuts* this) {
+    if (!EnLiftNuts_MinigameState(ENLIFTNUTS_MINIGAME_STATE_MODE_CHECK, ENLIFTNUTS_MINIGAME_STATE_HIDING)) {
+        EnLiftNuts_MinigameState(ENLIFTNUTS_MINIGAME_STATE_MODE_SET, ENLIFTNUTS_MINIGAME_STATE_HIDING);
+        this->isFirstTimeHiding = true;
     }
-    if (this->actionFunc == func_80AE9F70) {
-        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimations, 15);
+    if (this->actionFunc == EnLiftNuts_IdleHidden) {
+        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimations, ENLIFTNUTS_ANIM_BOB);
     }
-    this->unk_354 = 0;
-    this->actionFunc = func_80AEB8A4;
+    this->frames = 0;
+    this->actionFunc = EnLiftNuts_StartHiding;
 }
 
-void func_80AEB8A4(EnLiftNuts* this, PlayState* play) {
-    if (this->unk_354 == 22) {
-        if (this->unk_356 == 1) {
+void EnLiftNuts_StartHiding(EnLiftNuts* this, PlayState* play) {
+    if (this->frames == 22) {
+        if (this->isFirstTimeHiding == true) {
             Message_StartTextbox(play, 0x27F6, &this->actor);
             this->textId = 0x27F6;
         }
-        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimations, 17);
-        this->actionFunc = func_80AEB934;
+        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimations, ENLIFTNUTS_ANIM_BURROW_HALF);
+        this->actionFunc = EnLiftNuts_Hide;
     }
-    this->unk_354++;
+    this->frames++;
 }
 
-void func_80AEB934(EnLiftNuts* this, PlayState* play) {
+void EnLiftNuts_Hide(EnLiftNuts* this, PlayState* play) {
     if (Animation_OnFrame(&this->skelAnime, 5.0f)) {
-        func_80AEB9E0(this, play);
+        EnLiftNuts_SpawnDust(this, play);
     }
 }
 
-void func_80AEB974(EnLiftNuts* this) {
-    s16 textId = this->textId;
-
-    switch (textId) {
-        default:
-            this->unk_1E4 = 0;
-            break;
-
+void EnLiftNuts_SetEyeTexIndex(EnLiftNuts* this) {
```

Comment:
Currently, functions like this are split between `XXX_Blink` and `XXX_UpdateEyes`. These should probably be consolidated into 1, but for now I'd name this function with one of these two options